src/Firehose.Web/Authors/JaimeHernandez.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ServiceModel.Syndication;
+using Firehose.Web.Infrastructure;
+
+namespace Firehose.Web.Authors
+{
+    public class JaimeHernandez : IAmACommunityMember
+    {
+        public string FirstName => "Jaime";
+        public string LastName => "Hernandez";
+        public string StateOrRegion => "Santiago, Chile";
+        public string TwitterHandle => "HsJhernandez";
+        public string EmailAddress => "jhernandez.hs@icloud.com";
+        public string ShortBioOrTagLine => "Systems analyst developing web systems and mobile applications" ;
+        public string GravatarHash => "35939afef20aac1680fc71bbd6d9c214";
+        public Uri WebSite => new Uri("http://lawebdelprogramador.cl/bloglawebdelprogramador/");
+
+        public IEnumerable<Uri> FeedUris
+        {
+            get { yield return new Uri("hhttp://lawebdelprogramador.cl/bloglawebdelprogramador/feed/"); }
+        }
+
+        public string GitHubHandle => "lawebdeprogramador";
+
+        public bool Filter(SyndicationItem item) =>
```

Comment:
Your feed doesn't seem to have any categories, so you should probably only filter by title